### DISCUSSION TITLE
Where does faster-whisper downloads the model ?

### DISCUSSION THREAD

**Discussion by shrangideqode** [SYSTRAN/faster-whisper#1173] [General]
When initialising the whisper model
`model = WhisperModel("distil-large-v3", device="cuda", compute_type='float16')`
What path does it download the model ?
How can i download the model beforehand so that while initialising it does not download and only load the model.
I want to download the model in Docker image so that i start my application without waiting for it to downlaod.

**Comment by MahmoudAshraf97** [comment]
it's downloaded from huggingface to the huggingface cache dir

**Comment by hmf** [comment]
If you are in Linux that would be in (`user` is your user name):

```
/home/user/.cache/huggingface/hub/
```

For a list of model names see:

https://github.com/SYSTRAN/faster-whisper/blob/97a4785fa13d067c300f8b6e40c4381ad0381c02/faster_whisper/utils.py#L12

HTHs
